src/components/GameInfo.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+/**
+ * Type properties for Gameinfo Component
+ */
+type GameInfoProps = {
+  winner: string | null;
+  stepNumber: number;
+  currentPlayer: string;
+  dispatch: React.Dispatch<GameStateAction>;
+  history: GameHistory;
+  MovesButtons: Array<JSX.Element>;
+};
+
+/**
+ * Render Game Info and Game toolbar buttons
+ * @param param0
+ * @returns
+ */
+const GameInfo: React.FC<GameInfoProps> = ({
+  winner,
+  stepNumber,
+  currentPlayer,
+  history,
+  dispatch,
+  MovesButtons,
+}) => {
+  /**
+   * Determine gameover state
+   * @returns BoOlean
+   */
+  const isGameOver = () => history.length - 1 === 9 && winner === '';
```

Comment:
I'd define this outside of React and accept history and winner as args. That'll make this function more manageable and easier to test.